graphix/parameter.py

```diff
@@ -0,0 +1,921 @@
+    @property
+    def symbol(self) -> str:
+        return "conj"
+
+    def eval(self, children: list[complex]) -> complex:
+        return children[0].conjugate()
+
+    def simplify(self) -> Expression | complex:
+        simpl = super().simplify()
+        if isinstance(simpl, Conjugate):
+            return simpl.operand.conjugate()
+        return simpl
+
+    def conjugate(self) -> Expression | complex:
+        return self.operand
+
+
+class Associativity(Enum):
+    No = 0
+    Left = 1
+    Right = 2
+
+
+class Binary(Compound):
+    def __init__(self, lhs, rhs):
+        super().__init__((lhs, rhs))
+
+    @classmethod
+    def from_iterable(cls, operands: typing.Iterable[Expression]):
+        it = iter(operands)
+        lhs = next(it)
+        rhs = next(it)
+        return cls(lhs, rhs)
+
+    @property
+    def lhs(self) -> Expression:
+        return self[0]
+
+    @property
+    def rhs(self) -> Expression:
+        return self[1]
+
+    @property
+    def symbol(self) -> str: ...
+
+    @property
+    def precedence(self) -> int: ...
+
+    @property
+    def associativity(self) -> Associativity: ...
+
+    def format(self, precedence: int, target: list[str]):
+        need_parentheses = precedence > self.precedence
+        if need_parentheses:
+            target.append("(")
+        if self.associativity == Associativity.Left:
+            left_precedence = self.precedence
+        else:
+            left_precedence = self.precedence + 1
+        format_expr(self.lhs, left_precedence, target)
+        target.append(" ")
+        target.append(self.symbol)
+        target.append(" ")
+        if self.associativity == Associativity.Right:
+            right_precedence = self.precedence
+        else:
+            right_precedence = self.precedence + 1
+        format_expr(self.rhs, right_precedence, target)
+        if need_parentheses:
+            target.append(")")
+
+
+def add_linear_combination_terms(
+    terms: dict[Expression | complex, complex], coef: complex, expr: Expression | complex
+) -> None:
+    if isinstance(expr, Add):
+        add_linear_combination_terms(terms, coef, expr.lhs)
+        add_linear_combination_terms(terms, coef, expr.rhs)
+    elif isinstance(expr, Sub):
+        add_linear_combination_terms(terms, coef, expr.lhs)
+        add_linear_combination_terms(terms, -coef, expr.rhs)
+    elif isinstance(expr, Mul):
+        coef_others = expr.separe_coefficient()
+        if coef_others is None:
+            terms[expr] = terms.get(expr, 0) + coef
+        else:
+            mul_coef, others = coef_others
+            add_linear_combination_terms(terms, coef * mul_coef, others)
+    elif isinstance(expr, Minus):
+        add_linear_combination_terms(terms, -coef, expr.operand)
+    elif isinstance(expr, complex):
+        terms[None] = terms.get(None, 0) + coef * expr
+    else:
+        terms[expr] = terms.get(expr, 0) + coef
+
+
+def syntactic_order_key(key: Expression | complex | None) -> Any:
```

Comment:
Please import `typing`. Additionally, please do not use `Any` if possible as it ruins type annotations.